marklogic-data-hub-central/ui/src/components/load/new-load-dialog/new-load-dialog.test.tsx

```diff
@@ -102,4 +102,36 @@ describe('New/edit load data configuration', () => {
     expect(queryByText('Source Type')).not.toBeInTheDocument();
   });
 
+  test('Verify name is required for form submission and error messaging appears', async () => {
+    const { debug, baseElement, queryByText, getByText} = 
+    render(
+      <BrowserRouter><NewLoadDialog newLoad={true}
+      title={'New Loading Step'}
+      setNewLoad={() => {}}
+      createLoadArtifact={() => {}}
+      stepData={{}}
+      canReadWrite={true}
+      canReadOnly={false}/></BrowserRouter>
+    );
+    const nameInput = baseElement.querySelector('#name');
+
+    // error should not appear before anything is touched
+    expect(queryByText('Name is required')).toBeNull();
+
+    fireEvent.click(getByText('Save'));
+
+    // message should appear when save button is clicked
+    expect(queryByText('Name is required')).toBeInTheDocument();
+
+    // enter in a name and verify message disappears
+    await wait(() => {
+      if(nameInput){
+        fireEvent.change(nameInput, { target: {value: 'testLoadStep'} });
+      }
+    }); 
+    expect(nameInput).toHaveValue('testLoadStep');
+    
+    expect(queryByText('Name is required')).toBeNull();
```

Comment:
This check was failing for me locally, I was able to get it to work by updating it to the following:
```
    fireEvent.change(nameInput, { target: {value: 'testLoadStep'} });
    await wait(() => {
      expect(nameInput).toHaveValue('testLoadStep');
    }); 
    expect(queryByText('Name is required')).toBeNull();
```

Reply:
I was also able to reproduce on my local system.  But that is extremely strange, because I swear we both sat there and saw it pass yesterday.  In any event, it's fixed now.